agent/config/config.go

```diff
@@ -341,6 +344,16 @@ func environmentConfig() (Config, error) {
 		seelog.Debugf("Setting instance attribute %v: %v", attributeKey, attributeValue)
 	}
 
+	var additionalLocalRoutes []cnitypes.IPNet
+	additionalLocalRoutesEnv := os.Getenv("ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES")
+	if additionalLocalRoutesEnv != "" {
+		err := json.Unmarshal([]byte(additionalLocalRoutesEnv), &additionalLocalRoutes)
+		if err != nil {
+			seelog.Warnf("Invalid format for ECS_AWSVPC_ADDITIONAL_LOCAL_ROUTES, expected a json array of CIDRs: %v", err)
```

Comment:
Why is `warn` the right severity here?

Reply:
We're using `warn` for all of them right now.  I'd be happy to change all of them to `error` if you'd like.

Reply:
Actually, we're using `warn` for both non-fatal conditions (actually should be warnings) and for parsing errors (should be errors).  I'll change the ones that actually cause the agent to exit to `error`.

Reply:
Thanks, that's pretty much what I was getting at. If `NewConfig()` returns an error, agent panics. We should probably log the condition that cause the panic at an appropriately high severity.